packages/open-payments/src/client/grant.test.ts

```diff
@@ -10,23 +24,98 @@ describe('grant', (): void => {
     openApi = await createOpenAPI(config.OPEN_PAYMENTS_AS_OPEN_API_URL)
   })
 
-  const axiosInstance = defaultAxiosInstance
-  const logger = silentLogger
-  const client = 'https://example.com/.well-known/pay'
+  const deps = {
+    axiosInstance: defaultAxiosInstance,
+    logger: silentLogger,
+    client: 'https://example.com/.well-known/pay'
+  }
+
+  describe('routes', () => {
+    const url = 'http://localhost:1000'
+    const accessToken = 'someAccessToken'
+
+    describe('request', () => {
+      test('calls post method with correct validator', async (): Promise<void> => {
+        const mockResponseValidator = ({ path, method }) =>
+          path === '/' && method === HttpMethod.POST
+
+        jest
+          .spyOn(openApi, 'createResponseValidator')
+          .mockImplementation(mockResponseValidator as any)
```

Comment:
Should we be clearing these mocks?
I'm guessing we can't just do `jest.clearAllMocks()` in an `afterEach` because of the `jest.mock('./requests'`

Reply:
Because we have `clearMocks: true` in `jest.config.js`, they are cleared after every test already

https://jestjs.io/docs/configuration#clearmocks-boolean